lib/Services/Application.vala

```diff
@@ -0,0 +1,124 @@
+/*
+ *  Copyright 2019 elementary, Inc. (https://elementary.io)
+ *
+ *  This program or library is free software; you can redistribute it
+ *  and/or modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 3 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General
+ *  Public License along with this library; if not, write to the
+ *  Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ *  Boston, MA 02110-1301 USA.
+ */
+
+[CCode (gir_namespace = "GraniteServices", gir_version = "1.0")]
+namespace Granite.Services {
+    /**
+     * Utilities for Applications
+     */
+    namespace Application {
```

Comment:
It seems like we moved from namespacing anything altogether in Granite. All recently merged classes are located in `Granite` rather than in sub namespaces. Do we still want that here?

Reply:
I think that we are moving from namespacing regarding the Gtk widgets but here it's an utility and Granite.Application is already a class unfortunately

Reply:
Ah okay, then it makes sense, but since the class name is `UnityLauncherEntry`, can't just put that into `Services`?

Reply:
UnityLauncherEntry is the private DBus interface, the static functions aren't in a class :slightly_smiling_face: